tests/integration/catalogue/test_category.py

```diff
@@ -44,6 +45,13 @@ def test_unicode_slug(self):
             self.assertEqual(root_category.slug, u'vins-français')
             self.assertEqual(child_category.slug, u'château-dyquem')
 
+    def test_url_caching(self):
```

Comment:
Do we need to have memcached running to test this? Should be possible and more specific just to mock the `cache.set()` call and check that the right data is being saved to the cache.

Reply:
Yes, because default cache backend does not serialize and deserialize data as well as don't write it to the storage. One of of our projects, category slug, deserialized from the cache, has `bytes` type, not `str`, which is intended to verify by this change.

Reply:
Can't we use the default (locmem) or file based cache then? 

The reason I don't think adding memcached to the test dependencies is advisable is that memchached requires additional system-level libraries (python-dev, libmemcached-devel etc) which will make life that much harder for anyone who wants to set it up locally.